Website/src/rendering/src/assets/css/components/Speakers/SpeakersGrid.css

```diff
@@ -1,59 +1,38 @@
-.section__speakers__grid {
-  @apply mx-auto grid grid-cols-1 gap-5;
-  @apply sm:grid-cols-2;
-  @apply md:grid-cols-3;
-  @apply lg:grid-cols-4;
-  @apply xl:grid-cols-5;
-
-  margin-top: 20px;
-  margin-bottom: 40px;
-
-  &__speaker {
-    @apply box-border col-span-1 relative;
-
-    width: 100px;
-    min-width: 260px;
-    padding: 10px;
-    margin: 10px;
-
-    &:hover img {
-      box-shadow: 0 0 10px rgba(23, 23, 23, 0.45);
-    }
-
-    img {
-      @apply w-full;
-    }
+.section__speakers {
+  .section__content__title {
+    @apply text-center text-blue normal-case;
+  }
 
-    .speaker__image {
-      @apply w-full bg-none;
-    }
+  .section__speakers__grid {
+    @apply mx-auto grid grid-cols-1 gap-5 mt-5 mb-10;
+    @apply sm:grid-cols-2;
+    @apply md:grid-cols-3;
+    @apply xl:grid-cols-4 gap-10;
```

Comment:
To have the `gap-10` apply only for the xl breakpoint, it must be prefixed with `xl:`. With the current CSS, all breakpoints have a `gap-10`.